### PR TITLE
GEODE-5613: Add winrm-cli utility to apachegeode-build image

### PR DESCRIPTION
**Pull request by jdeppe-pivotal**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

**Comment by jdeppe-pivotal** [comment]
@rhoughton-pivot @smgoller For your review.

File: ci/docker/Dockerfile [hunk 1]
```diff
@@ -57,6 +57,7 @@ RUN chmod +x /usr/local/bin/tini \
     python3-setuptools \
     unzip \
     vim \
+    make \
```
Comment (rhoughton-pivot) [inline ci/docker/Dockerfile]: Only because the rest already is, can we alphabetize the list?

File: ci/docker/Dockerfile [hunk 2]
```diff
@@ -71,6 +72,9 @@ RUN chmod +x /usr/local/bin/tini \
   && mv /opt/selenium/chromedriver /opt/selenium/chromedriver-$CHROME_DRIVER_VERSION \
   && chmod 755 /opt/selenium/chromedriver-$CHROME_DRIVER_VERSION \
   && ln -fs /opt/selenium/chromedriver-$CHROME_DRIVER_VERSION /usr/bin/chromedriver \
+  && git clone https://github.com/masterzen/winrm-cli \
+  && (cd winrm-cli; GOPATH=$PWD PATH=$PATH:$PWD/bin make) \
+  && cp winrm-cli/bin/winrm /usr/local/bin/ \
```
Comment (rhoughton-pivot) [inline ci/docker/Dockerfile]: If the winrm binary is statically linked or owns all its dependencies, can we clean up the cloned `winrm` directory when we're done copying out the executable?

File: ci/docker/Dockerfile (hunk 1 above)
Comment (jdeppe-pivotal) [reply ci/docker/Dockerfile]: Done

File: ci/docker/Dockerfile (hunk 2 above)
Comment (jdeppe-pivotal) [reply ci/docker/Dockerfile]: It already is a few lines further down - `rm -rf /tmp/work`

Comment (rhoughton-pivot) [reply ci/docker/Dockerfile]: Is it safe to rm the workdir of the image we are building?

Comment (jdeppe-pivotal) [reply ci/docker/Dockerfile]: Yes.

**Pull request merged**